app/src/main/java/cc/pogoda/mobile/meteosystem/dao/AvailableParametersDao.java

```diff
@@ -63,7 +63,9 @@ public AvailableParametersWeb getAvaliableParamsByStationName(String name) {
         try {
             t.join();
 
-            out = response.body();
+            if (response != null) {
```

Comment:
@rwoloszyn I had to add this check as during tests with very limited comm speed (~14k4 bps) the app were crashed because of very slow (or no response) from the APi